app/views/rooms/show.html.erb

```diff
@@ -0,0 +1,61 @@
+<%= provide(:title, "Show") %>
+<div class="mt-4">
+    <% @rooms.each do |room| %>
+        <div class="col-10">
+            <%= image_tag room.images.first.image, class: "d-block mx-auto img-fluid w-50" %>
+        </div>
+    <% end %>
+</div>
+<div class="ml-4 mt-4">   
+        <h1><%= @room.listing_name %></h1>  
+        <h6><%= @room.address %></h6>
+</div>
+<hr width="1200" align="center">
+<div class="ml-4 mt-4">
+    <h4>About this listings</h4>
+    <h6><%= @room.summery %></h6>
+</div>
+<hr width="1200" align="center">
+<div class="ml-4 mt-4">
+    <h4>Amenities</h4>
+</div>
+<hr width="1200" align="center">
+<div id="carouselExampleFade" class="offset-2 col-8 offset-2 carousel slide carousel-fade" data-ride="carousel">
+    <div class="carousel-inner">
+        <div class="carousel-item active">
+            <% @rooms.each do |room| %>
+                <%= image_tag room.images.first.image, class: "d-block mx-auto img-fluid w-100" %>
+            <% end %>
+        </div>
+    <a class="carousel-control-prev" href="#carouselExampleFade" role="button" data-slide="prev">
+        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+        <span class="sr-only">Previous</span>
+    </a>
+    <a class="carousel-control-next" href="#carouselExampleFade" role="button" data-slide="next">
+        <span class="carousel-control-next-icon" aria-hidden="true"></span>
+        <span class="sr-only">Next</span>
+    </a>
+</div>
```

Comment:
make sure the carousel is a fully functional carousel..

please do research in how to use the bootstrap carousel with rails active storage..

hint(its a forum thread in english):
https://stackoverflow.com/questions/38357072/bootstrap-carousel-in-rails/38357266